### ISSUE TITLE
Encountered an error when trying in the huggingface platform

### ISSUE THREAD

**Issue by Shimao-Zhang** [IDEA-Research/Grounded-Segment-Anything#191]
![image](https://user-images.githubusercontent.com/80098168/233697884-1f111b41-786e-469b-b196-1548f55cc006.png)
Does this output mean the model can't find the corresponding box in the picture?

**Comment by rentainhe** [comment]
Yes it means there's no object be detected, we will try to modify the information in the future version
